frontend/pages/reports/index.tsx

```diff
@@ -330,7 +330,9 @@ const ReportsIndex: NextPage = () => {
 
                 <MultiChoice
                   items={(meta.data?.systems || []).map(x => ({
-                    key: x.value,
+                    key: i18n?.exists(`metadata:${x.value}`)
+                      ? x.value + ' / ' + t(`metadata:${x.value}`)
```

Comment:
This could also be done as
```suggestion
                      ? `${x.value} / ${t(`metadata:${x.value}`)}`
```
Though now that I've written it out I'm not sure if this is as readable 😅

Reply:
No i think you are right, as i used template literals already on the row above, better to keep the uniformity and use them on this row also.